### DISCUSSION TITLE
is cypress supports, output txt file validation?

### DISCUSSION THREAD

**Discussion by SonamBaghel** [cypress-io/cypress#18001] [Questions and Help]
Hello there,
I need help on below thing,
My test application is having some input file and as a output it generates some txt file.
Now here using cypress I need to automate, i need to validate output file on basis on inputs provided.
is there any way which I can follow?..

**Comment by bahmutov** [comment]
What do you want to validate in the produced file? That it contains some specific text or words?

**Reply by SonamBaghel** [reply]
yeah I want to find some particular static words within the file and console dynamic values ..
